cores/arduino/Uart.cpp

```diff
@@ -91,10 +91,13 @@ void Uart::flush()
 void Uart::IrqHandler()
 {
   if (sercom->availableDataUART()) {
-    sercom->isFrameErrorUART();
-    uint8_t c=sercom->readDataUART();
-    if( !sercom->isUARTError()){
+    uint8_t c = sercom->readDataUART();
+    if (!sercom->isUARTError()) {
```

Comment:
I still think this condition is incorrect, please look at the data sheet again, not all error situations should result in us dropping the byte.